### PR TITLE
[#5513] Add client CLI to github action yaml file

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Add client CLI to github action yaml file.

### Why are the changes needed?

To make integration test run when CLI code changes happen.

Fix: #5513

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

N/A Unable to test locally.

### CODE REVIEW

**Comment by jerryshao** [comment]
I assume the "build.yml" should also be updated.

**Comment by jerryshao** [comment]
@mchades can you please help to review this PR.

File: .github/workflows/backend-integration-test.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ jobs:
               - api/**
               - bin/**
               - catalogs/**
+              - clients/cli/**
```
Comment (mchades) [inline .github/workflows/backend-integration-test.yml:28]: also need to add this to `.github/workflows/build.yml`

Comment (justinmclean) [reply .github/workflows/backend-integration-test.yml:28]: Done

File: .github/workflows/backend-integration-test.yml
Comment (mchades) [inline .github/workflows/backend-integration-test.yml:1]: I did not see integration testing in the CLI module, so should changes to the CLI module trigger backend CI?

Comment (justinmclean) [reply .github/workflows/backend-integration-test.yml:1]: I guess not, removing it.

**Review by mchades — APPROVED** [review]
LGTM.

**Pull request merged**